db/migrate/20220810131220_add_automated_activity_to_audits.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class AddAutomatedActivityToAudits < ActiveRecord::Migration[6.1]
+  # rubocop:disable Metrics/MethodLength
+  def up
+    add_column(
+      :audits,
+      :automated_activity,
+      :boolean,
+      default: false,
+      null: false
+    )
+
+    execute(
+      "UPDATE audits
+      SET automated_activity = TRUE
+      WHERE activity_type IN (
+        'description_change_validation_request_auto_closed',
+        'red_line_boundary_change_validation_request_auto_closed'
+      );"
+    )
+
+    execute(
```

Comment:
Updates the 'description updated' audit associated with the auto-closure of the description change request.